doozer

```diff
@@ -87,6 +111,7 @@ def print_version(ctx, param, value):
               help='Process Dockerfiles in ODCS mode. HACK for the time being.')
 @click.option('--load-disabled', default=False, is_flag=True,
               help='Treat disabled images/rpms as if they were enabled')
+@click.option('--local/--osbs', default=False, is_flag=True, help='--local to run in local-only mode, --osbs to run on build cluster (default)')
```

Comment:
Ok, so the reason for this was that (because of how the settings.yaml stuff works) you could set `local: true` in your settings and it would *always* run in local mode. So `--osbs` puts you back in regular mode. It was just easier to do that as a single option in click. We do that already with `--push/--no-push` which I realize makes more sense... but `--local/--no-local` is dumb.